### ISSUE TITLE
Stripes `monthly.json` demand series object is different from the rest

### ISSUE THREAD

**Issue by chienleng** [opennem/opennem#139]
JSON: https://data.opennem.org.au/v3/stats/au/all/monthly.json

<img width="856" alt="Screen Shot 2022-09-23 at 5 44 13 pm" src="https://user-images.githubusercontent.com/349944/191914251-c34679c4-b5f0-4d3e-8f8f-bcca598af26e.png">

for all `demand.energy` and `demand.market_value`, it's missing the `region` prop (FE uses this to filter out the flat series for each region. Also note the `code` prop is different for demand and non-demand series.

**Comment by simonhac** [comment]
would be good to standardise the properties. 

`code` seems like a poor name for an attribute.

**Comment by chienleng** [comment]
yep, I think `code` can be removed. use `fuel_tech` and `region`.
